index.js

```diff
@@ -426,35 +426,33 @@ class K8sExecutor extends Executor {
             url: `${this.podsUrl}/${podName}/status`,
             method: 'GET',
             headers: { Authorization: `Bearer ${this.token}` },
-            https: { rejectUnauthorized: false },
-            retry: {
-                statusCodes: [200],
-                limit: this.maxAttempts,
-                calculateDelay: ({ computedValue }) => (computedValue ? this.retryDelay : 0)
-            },
-            hooks: {
-                afterResponse: [this.scheduleStatusRetryStrategy]
-            }
+            https: { rejectUnauthorized: false }
         };
 
-        const resp = await this.breaker.runCommand(statusOptions);
+        try {
+            const resp = await request(statusOptions);
+
+            logger.debug(`Build ${buildId} pod response: ${JSON.stringify(resp.body)}`);
+            if (resp.statusCode !== 200) {
```

Comment:
Probably can remove this since the catch block does the same

Reply:
but catch block is for error isn't it? If API returns other than 200 status?